packages/core/src/agent/message-list/message-list.ts

```
@@ -40,6 +40,7 @@ import type {
 } from './state';
 import type { AIV5Type, AIV5ResponseMessage, AIV6Type, MessageInput, MessageListInput } from './types';
 import { ensureGeminiCompatibleMessages } from './utils/provider-compat';
+import { normalizeModelOutput } from '../../loop/workflows/agentic-execution/normalize-model-output';
```

ESLint

[error] `../../loop/workflows/agentic-execution/normalize-model-output` import should occur before import of `../../tools/payload-transform`

(import/order)